_posts/2017-04-24-get-electrified-3.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+comments: true
+
+title: Get Electrified 3
+excerpt: Tinkering with Digital Electronics!
+author: Mayanka Medhe
+category: [Informative]
+tags: [Digital Electronics, Adder, MUX]
+
+assets-dir: assets/get-electrified-3
+header-img: "assets/get-electrified-3/cover.jpg"
+---
+
+Electronics club conducted its last electrified session on 18th March, 2017. It was totally awesome as we got an opportunity to make a half adder, full adder, subtractor and priority encoder. Along with that we also learnt some cool stuff about digital circuits. 
+
+ 
+
+**Where do we begin from to make anything as simple as an adder?**
+
+We basically use a combinational circuit. Not only an adder but we can even make a subtractor, priority encoder, MUX and what not!
+
+Combinational circuit is basically used for making those circuits which depends on the current state.
+
+![Combinational Logic Circuits]({{ site.baseurl }}/{{ page.assets-dir }}/image_0.png)
+
+**So, let us now begin with how exactly do these combinational circuits function:**
+
+** Adder:**
+
+As you can guess an adder simply add two numbers. But why do we need a complicated circuit to just add numbers? Actually, adders are used in the arithmetic logic units in many computers and other kinds of processors.
+
+There are two kinds of adder:
+
+**1)**      **Half adder**
+
+**2)**      **Full adder**
+
+The major difference between a half adder and a full adder is that a full-adder has three inputs and two outputs where as a half adder has only two inputs and two outputs. In a full adder, the first two inputs are A and B and the third input is an input carry.
+
+The circuit diagram and truth table of a half adder and full adder is shown below:
+
+![Half Adder]({{ site.baseurl }}/{{ page.assets-dir }}/image_1.png)
+
+![Full Adder]({{ site.baseurl }}/{{ page.assets-dir }}/image_2.png)                                                                     
+
+	                      
+
+**Subtractor:**
```

Comment:
If you make the Adder a heading, do the same here and a few more places below. Basically any subheading worthy bold line should have consistency.